apps/ejabberd/src/ejabberd_rdbms.erl

```diff
@@ -34,7 +34,7 @@
 start() ->
     compile_odbc_type_helper(),
     %% Check if ejabberd has been compiled with ODBC
-    case catch ejabberd_odbc_sup:module_info() of
+    case catch mongoose_rdbms_sup:module_info() of
         {'EXIT', {undef, _}} ->
             ?INFO_MSG("ejabberd has not been compiled with relational database support. Skipping database startup.", []);
```

Comment:
Since Elvis is complaining about this line, I suggest changing the log message a bit so that is starts with "MongooseIM".